fickling/fickle.py

```diff
@@ -706,6 +708,12 @@ def has_non_setstate_call(self) -> bool:
         object.__setstate__"""
         return bool(self.properties.non_setstate_calls)
 
+    def check_safety(self):
+        raise WrongMethodError
+
+    def is_likely_safe(self):
+        raise WrongMethodError
```

Comment:
To reflect the suggested changes in `WrongMethodError`
```suggestion
    def check_safety(self):
        raise WrongMethodError("This method has been removed. Use fickling.analysis.check_safety() on the Pickled object instead")

    def is_likely_safe(self):
        raise WrongMethodError("This method has been removed. Use fickling.is_likely_safe() on the pickle file instead")
```